.erda/migrations/pipeline/20240221-pipeline-crons-add-softdelete.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE `pipeline_crons` ADD COLUMN `soft_deleted_at` bigint(20) NOT NULL DEFAULT '0' COMMENT '软删除';
```

Comment:
add `after xxx` to ensure new field location.

Reply:
Done.